extension/chrome/elements/pgp_pubkey.ts

```diff
@@ -30,6 +30,7 @@ Catch.try(async () => {
   const minimized = uncheckedUrlParams.minimized === true;
 
   const { keys: pubs } = await openpgp.key.readArmored(armoredPubkey);
+  const isUsableButExpired = await Pgp.key.usableButExpired(pubs[0]) || undefined;
```

Comment:
Why is the `|| undefined` used?

Reply:
oh, yes, I will remove it.

I needed it when I was saving `Boolean` to `Contact` model in our IndexedDB.
Thanks!